### PR TITLE
Fix install:git, make:database, update:env commands

### PR DESCRIPTION
**Pull request by dexm238**
Add necessary `optionMap` to some commands

### CODE REVIEW

File: src/Commands/Databases/Make.php [hunk 1]
```diff
@@ -27,6 +28,7 @@ public function configure()
             ->addArgument('server', InputArgument::REQUIRED, 'The id of the server to create the database on.')
             ->addOption('user', null, InputOption::VALUE_REQUIRED, 'The username of an (optional) database user to create.', null)
             ->addOption('password', null, InputOption::VALUE_REQUIRED, 'The password of the database user. Required with "user".', null)
+            ->addOption('name', null, InputOption::VALUE_REQUIRED, 'The name of the database user.', null)
```
Comment (svenluijten) [inline src/Commands/Databases/Make.php]: This is actually the name of the database to create, not the name of the database user 🙂

File: src/Commands/Env/Update.php [hunk 2]
```diff
@@ -10,6 +10,13 @@
 
 class Update extends BaseCommand
 {
+    /**
+     * @var array
+     */
+    protected $optionMap = [
```
Comment (svenluijten) [inline src/Commands/Env/Update.php:16]: Not required when the keys are the same as the values.

File: src/Commands/Projects/InstallGit.php [hunk 3]
```diff
@@ -10,6 +10,14 @@
 
 class InstallGit extends BaseCommand
 {
+    /**
+     * @var array
+     */
+    protected $optionMap = [
```
Comment (svenluijten) [inline src/Commands/Projects/InstallGit.php:16]: Not required when the keys are the same as the values.

File: src/Commands/BaseCommand.php [hunk 4]
```diff
@@ -87,7 +87,7 @@ protected function getFileContent(InputInterface $input, $option)
             return $filename;
         }
 
-        if ($filename && ftell(STDIN) === 0) {
+        if ($filename && ftell(STDIN)) {
```
Comment (svenluijten) [inline src/Commands/BaseCommand.php]: Why this? `ftell()` returns the position of the pointer. I'd be happy to use `ftell(STDIN) !== false` if you were looking for more type-safety here.

**Review by svenluijten — CHANGES_REQUESTED** [review]
Thanks! I've added some comments/questions as a review 😄

File: src/Commands/BaseCommand.php (hunk 4 above)
Comment (dexm238) [reply src/Commands/BaseCommand.php]: I agree)

File: src/Commands/Env/Update.php (hunk 2 above)
Comment (dexm238) [reply src/Commands/Env/Update.php:16]: `fillData()` returns an empty array in this case...

**Comment by svenluijten** [comment]
Thank you!

**Pull request merged**